find_courses.py

```diff
@@ -66,13 +64,13 @@ def main():
                 i = i + 1
             is_running = False
         else:
-            print("\nNo courses were found that match your input.")
-            try_again = raw_input("\nWould you like to try again? Enter Y to try again or any other key to leave. ")
+            print(NO_COURSES_MESSAGE)
+            try_again = raw_input(TRY_AGAIN_MESSAGE)
             if try_again != "Y":
```

Comment:
this "Y" should be in the consts file too. We can have them called keys. like TRY_AGAIN_NO_COURSES_KEY. something like that